ext/openssl/tests/001.phpt

```diff
@@ -62,16 +62,22 @@ openssl_pkey_free($loaded_key);
 echo "OK!\n";
 
 ?>
---EXPECT--
+--CLEAN--
+<?php
+$key_file_name = __DIR__ . DIRECTORY_SEPARATOR . '001-tmp.key';
+@unlink($key_file_name);
+?>
+--EXPECTF--
 Creating private key
 Export key to file
 bool(true)
 Load key from file - array syntax
+
+Deprecated: Function openssl_pkey_free() is deprecated in %s on line %d
```

Comment:
we should probably remove it from the test if it's deprecated - again doesn't need to be done now.